ci-operator/config/openstack-k8s-operators/openstack-k8s-operators-ci/openstack-k8s-operators-openstack-k8s-operators-ci-main.yaml

```diff
@@ -49,6 +49,24 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.18-openshift-4.12
   to: ci-build-root-golang-1.19-sdk-1.31
+- build_args:
+  - name: SDK_VERSION
+    value: v1.31.0
```

Comment:
are we going to continue on sdk 1.31? should we double check with core team if we need to upgrade?
otherwise, we can also update that later if needed.

Reply:
After speaking with the devs we would like to stick with 1.31 for now as the update is not required and would rather do it in two stages, golang update and then sdk update as there are some breaking changes.